src/components/Menu.vue

```diff
@@ -156,16 +156,16 @@
           document.removeEventListener('click', documentClick);
         });
 
-        watch(isOpen.value, (isOpen, prevIsOpen) => {
-          nextTick(() => {
+        watch(isOpen, (isOpen, prevIsOpen) => {
+          setTimeout(() => {
             if (!prevIsOpen && isOpen) {
               openMenu();
             }
             if (prevIsOpen && !isOpen) {
               closeMenu();
             }
           })
-        }, {deep: true, immediate: true});
```

Comment:
I think `deep, immediate` is not required as, as `isOpen` is already reactive. Let me know if this wrong, But It still works 😃

Reply:
Seems fine to me, just learning vue myself.